### ISSUE TITLE
Windows 10 error

### ISSUE THREAD

**Issue by luminarious** [RazrFalcon/svgcleaner-gui#14]
I tried it out on Windows 10 build 17120 and got this strange error:
```
Process 'svgcleaner-cli' exit with error:
thread 'main' panicked at 'already borrowed: BorrowMutError', src\libcore\result.rs:906:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by RazrFalcon** [comment]
Can you post an input SVG file?

PS: it's not a Windows error.

**Comment by luminarious** [comment]
Hope this helps
[svg.zip](https://github.com/RazrFalcon/svgcleaner-gui/files/1827228/svg.zip)

**Comment by RazrFalcon** [comment]
That's was already resolved. I'll make a new release build soon.

**Comment by RazrFalcon** [comment]
Was fixed in v0.9.4.
